### DISCUSSION TITLE
_count doesn't update in many to many relation

### DISCUSSION THREAD

**Discussion by Sakkhor909** [prisma/prisma#12294] [Q&A]
This is the model :
```
model Category {
  id        String              @id @default(cuid())
  name       String              @unique
  createdAt DateTime            @default(now())
  brands Brand[]
}

model Brand {
  id         String              @id @default(cuid())
  name       String              @unique
  createdAt  DateTime            @default(now())
  categories   Category[] 
}
```
I am showing how many brands in a category by pulling the data like this 
```
   const Data = await prisma.category.findMany({
        include: {
          _count: {
            select: { brands: true }
          }
        }
      });
```
It works fine. It shows how many brands are in a category.
When I try to delete a brand like this;
```
 prisma.brand
      .delete({
        where: {
          id: ID
        }
      })
```
The brand gets deleted but the category list doesn't update. It still includes that brand in _count.
I have deleted the same way in one-to-many relation where _count updated. How do I do it in many to many?

**Comment by Sakkhor909** [comment]
Help please

**Comment by janpio** [comment]
Is the `Brand` gone after you deleted it? 
And `_count` is still returning as if it was note deleted?
Any special configuration for Prisma? What version?

**Reply by Sakkhor909** [reply]
Yes, The Brand is gone. `_count` returns the number as if the brand did not delete at all. I am using planet-scale for the database. Prisma version 3.11.0

**Reply by janpio** [reply]
Can you please turn this into an issue and provide a reproduction we can run ourselves? Thanks.

**Reply by Sakkhor909** [reply]
I have turned this into an issue but didn't understand reproduction?

**Reply by janpio** [reply]
Can you add a link to the issue here?
A reproduction is a piece of code or even full project in a GitHub repository we can use to get the same experience as you. It helps us to get to a reproduction of the problem faster and avoids us having to try to recreate what you are doing and experiencing manually.

**Reply by Sakkhor909** [reply]
Here is the issue link https://github.com/prisma/prisma/issues/12557

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
